packages/cli/test/dev/integration-4.test.ts

```diff
@@ -248,21 +248,25 @@ test(
   })
 );
 
-// https://linear.app/vercel/issue/ZERO-2919/investigate-platform-errors-and-restore-skipped-tests
-// eslint-disable-next-line jest/no-disabled-tests
-test.skip(
+test(
   '[vercel dev] Middleware with error in function handler',
   testFixtureStdio('middleware-error-in-handler', async (testPath: any) => {
-    await testPath(500, '/', /EDGE_FUNCTION_INVOCATION_FAILED/);
+    await testPath(
+      500,
+      '/',
+      /INTERNAL_SERVER_ERROR|MIDDLEWARE_INVOCATION_FAILED/g
```

Comment:
These assertions check two possible options because a deployed test of this scenario produces one result that the dev server can't currently replicate.

I spent some time digging into what the difference is between this and the next test scenarios that produces 2 different error codes in a deployment (`INTERNAL_SERVER_ERROR` and `MIDDLEWARE_INVOCATION_FAILED`), but only one in the dev server (`MIDDLEWARE_INVOCATION_FAILED`). I need to move on to other things.

If anyone wants to pair on digging deeper here, let me know!